source/modules/MEI.visual.xml

```diff
@@ -1615,13 +1615,18 @@
         </datatype>
       </attDef>
       <attDef ident="lines.color" usage="opt">
-        <desc>Captures the colors of the staff lines. The value is structured; that is, it should
-          have the same number of space-separated RGB values as the number of lines indicated by the
-          lines attribute. A line can be made invisible by assigning it the same RGB value as the
-          background, usually white.</desc>
+        <desc>Captures the colors of the staff lines.</desc>
         <datatype maxOccurs="unbounded">
           <rng:ref name="data.COLOR"/>
         </datatype>
+        <remarks>
+          <p>The value is structured; that is, it should contain a single color value or have 
+            the same number of space-separated values as the number of lines indicated by the
+            lines attribute. The first value then applies to the lowest line of the staff. 
+            A line can be made invisible by assigning it the same color as the
```

Comment:
I think we should strike this last sentence since `data.COLOR` allows alpha-channel colours, so it is possible to make a fully transparent ("invisible") line with the values in this attribute.